README.rst

```diff
@@ -82,70 +84,82 @@ PyPI wheels (pip)
 
 Lightning plugins can be installed using ``pip`` as follows
 
-.. code-block:: console
+.. code-block:: bash
 
-    $ pip install pennylane-lightning
+    pip install pennylane-lightning
 
 The above command will install the Lightning-Qubit plugin (the default since it is most broadly supported).
 In order to install the Lightning-GPU and Lightning-Kokkos (OpenMP) backends, you can respectively use the following commands:
 
-.. code-block:: console
+.. code-block:: bash
 
-    $ pip install pennylane-lightning[gpu]
-    $ pip install pennylane-lightning[kokkos]
+    pip install pennylane-lightning[gpu]
+    pip install pennylane-lightning[kokkos]
 
 
 Install from source
 ===================
 
 To build Lightning plugins from source you can run
 
-.. code-block:: console
+.. code-block:: bash
 
-    $ PL_BACKEND=${PL_BACKEND} pip install pybind11 pennylane-lightning --no-binary :all:
+    PL_BACKEND=${PL_BACKEND} pip install pybind11 pennylane-lightning --no-binary :all:
+    
+.. code-block:: bash
 
-where ``${PL_BACKEND}`` can be ``lightning_qubit`` (default), ``lightning_gpu`` or ``lightning_kokkos``.
+    PL_BACKEND=${PL_BACKEND} pip install pybind11 pennylane-lightning --no-binary :all:
+
+where ``${PL_BACKEND}`` can be ``lightning_qubit`` (default), ``lightning_gpu``,  ``lightning_kokkos``, or ``lightning_tensor``.
```

Comment:
Do we need quotation marks for ``lightning_qubit`` (default), ``lightning_gpu``, ``lightning_kokkos``, or ``lightning_tensor``?

Reply:
I think yes to make it easier to identify the backend names :thinking: